### ISSUE TITLE
two refresh buttons when playwright and python extensions are enabled

### ISSUE THREAD

**Issue by DetachHead** [microsoft/playwright-vscode#97]
![image](https://user-images.githubusercontent.com/57028336/154376563-dfdb37d8-af27-4b84-b3b1-a599b7350821.png)

**Comment by pavelfeldman** [comment]
Does the python button work for you?

**Comment by connor4312** [comment]
In the linked issue, there is a now [standard API for this](https://github.com/microsoft/vscode/issues/139737) which will be stable in our next release (<= 14 days from today)

**Comment by weyert** [comment]
Yeah, I am also seeing to Refresh buttons. If I press on it I am getting an error talking about pytests

**Comment by KotlinIsland** [comment]
Talking about getting an error message, I need to see the full-form of the message or I can't fix it!
